classes/form/FormBuilderVocabulary.inc.php

```diff
@@ -434,7 +434,6 @@ function _smartyFBVTextInput($params, &$smarty) {
 				case 'value':
 				case 'uniqId':
 					$smarty->assign('FBV_' . $key, $value); break;
-					break;
```

Comment:
Should I leave it as is now? Or take, we will take care of it at another commit?